scripts/generate-clientset.sh

```diff
@@ -63,20 +63,6 @@ if ${SILENT:-false}; then
   LOGGING_FLAGS=""
 fi
 
-tools=()
-for tool in client-gen deepcopy-gen informer-gen lister-gen conversion-gen; do
```

Comment:
might want to check they're all installed and in the path before trying to use them, just to exit early.

Reply:
done